routes/api/landing.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const router = express.Router();
+
+const City = require("../../models/city");
+
+router.get("/", function (req, res) {
+  City.find({}, function (err, doc) {
+    if (err) { 
+      res.status(400).json(err); 
+    }
+    var destinations = [];
+    var i;
+    //change loop limit if want more featured cities
+    var index = Math.floor(Math.random() * doc.length)
+    var headImg = doc[index].imgMain
+    var headCityName = doc[index].name
+    for (i = 0; i < 5; i++) {
+      let a = Math.floor(Math.random() * doc.length);
+
+      destinations.push({cityName:doc[a].name,
+                            picurl:doc[a].imgMain});
+    }
+    const result = {
+      header: {
+        cityName: headCityName,
+        headImg: headImg
+      },
+      featureDestination: destinations
+    }
+    res.send(result);
+     
+      /* **If repeating cities is concern, use shuffle **
+      doc = shuffle(doc);
+      for(i = 0; i < 5; i++){
+          destinations.push(doc.pop());
+      }
+      res.send(destinations);
+      */
+    })
+    
+
+  //fisher yates shuffle of cities
+function shuffle(arr) {
```

Comment:
is this function needed?

Reply:
Will be done in different story. (Function will be refactored)